eng/cake/dotnet.cake

```diff
@@ -456,23 +453,39 @@ void StartVisualStudioForDotNet6(string sln = null)
             sln = "./Microsoft.Maui-mac.slnf";
         }
     }
-    if (isCIBuild)
+    else
+    {
+        var vsLatest = 
+            VSWhereLatest(new VSWhereLatestSettings { IncludePrerelease = includePrerelease, })?.CombineWithFilePath("./Common7/IDE/devenv.exe");
```

Comment:
Does this logic actually need to live in the powershell script? If we think people will use powershell, we should put it there. Alternatively, we could delete the powershell script and always use cake?

Reply:
Deleted the `ps1` script and added the `debug` switch for binlogs. I noticed you were basically setting all the same variables already in cake and it all seems to work.  I also set it up so the pack will run conditionally. If it hasn't been ran before it'll run but it won't run if it already sees `MAUI` packs. If you want to force another run you can pass in `pack=t`.